Scripts/create_emme_project.py

```diff
@@ -118,47 +118,39 @@ def create_emme_project(args):
     parser.add_argument(
         "--log-level",
         choices={"DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"},
-        default=config.LOG_LEVEL,
     )
     parser.add_argument(
         "--log-format",
         choices={"TEXT", "JSON"},
-        default=config.LOG_FORMAT,
     )
     parser.add_argument(
         "--project-name",
         type=str,
-        default=config.PROJECT_NAME,
         help="Name of LEM project. Influences name of database directory"),
     parser.add_argument(
         "--submodel",
         type=str,
-        default=config.SUBMODEL,
         help="Name of submodel, used for choosing appropriate database dimensions"),
     parser.add_argument(
         "--emme-path",
         type=str,
-        default=config.EMME_PROJECT_PATH,
         help="Filepath to folder where EMME project will be created"),
     parser.add_argument(
         "--number-of-emme-scenarios",
         type=int,
-        default=config.NUMBER_OF_EMME_SCENARIOS,
         help="Number of scenarios in the emmebank"),
     parser.add_argument(
         "-s", "--separate-emme-scenarios",
         action="store_true",
-        default=config.SEPARATE_EMME_SCENARIOS,
         help="Using this flag enables saving network time-period specific results in separate EMME scenarios."),
     parser.add_argument(
         "--first-scenario-id",
         type=int,
-        default=config.FIRST_SCENARIO_ID,
         help="First EMME project scenario ID"),
+    parser.set_defaults(
+        **{key.lower(): val for key, val in config.items()})

```

Comment:
This could be simplified if we change the parameter names to lower keys in `config`.

Reply:
Maybe it's better to leave this as it is. It's not that much extra code and can avoid some potentially hard to find errors if someone mistypes the case of the key.